sklearn/utils/validation.py

```diff
@@ -8,35 +8,18 @@
 # License: BSD 3 clause
 import warnings
 import numbers
+from inspect import getargspec
 
 import numpy as np
 import scipy.sparse as sp
 
 from ..externals import six
 from ..utils.fixes import signature
+from ..exceptions import (DataConversionWarning, NonBLASDotWarning,
+                          NotFittedError)
```

Comment:
No... all 3 are used in this file! (`NotFittedError` is raised in `check_is_fitted`, `DataConversionWarning` raised inside `check_array` and the `NonBLASDotWarning` is silenced in the next few lines.)